zoom_monitor.py

```diff
@@ -148,20 +148,10 @@ def _open_participants_list(self):
         except NoSuchElementException:
             pass  # We need to open it.
 
-        # Sometimes, the controls to open the participants list get hidden
-        # (for example, when someone stops sharing their screen, maybe?). In
-        # order to make the controls visible again, move the mouse. We move
-        # the mouse by finding a DOM component that should always be on the
-        # screen, and then moving the mouse to two locations near it.
-        main_element_id = "wc-content"
-        self._wait_for_element(By.ID, main_element_id)
-        main_element = self._driver.find_element(By.ID, main_element_id)
-        ActionChains(self._driver)\
-            .move_to_element_with_offset(main_element, 0, 0)\
-            .move_to_element_with_offset(main_element, 1, 1)\
-            .perform()
-
-        self._wait_for_element(By.CLASS_NAME, "SvgParticipantsDefault")
+        element = self._wait_for_element(
+            By.XPATH, "//*[contains(@class, 'SvgParticipants')]")
+        hovering = element.get_attribute("class") == "SvgParticipantsHovered"
```

Comment:
and maybe this needs a comment, too. 
> If the class is `SvgParticipantsDefault`, we need to click it twice for reasons we're not sure of. However, if the class is `SvgParticipantsHovered`, clicking it twice will open it and then close it again. So, keep track of which situation we're in, so we can click the proper number of times.

Reply:
This comment might be unnecessary since we added the comment explaining this when the double click happens.